### ISSUE TITLE
Broken links with Mathjax \eqref

### ISSUE THREAD

**Issue by DavidPowell** [hakimel/reveal.js#2222]
I have Mathjax equations within my slides, and I have enabled automatic equation with Mathjax autonumbering. I then create an equation label with `\label` and refer to it later with `\eqref`

For example, I create the following slide:

````latex
<section>
	\begin{equation} \label{straightline}
	y = mx+c
	\end{equation}
	
	Equation \eqref{straightline} is a straight line
</section>
````

The equation is numbered correctly, and the link to it has the correct number as well.

However, mathjax turns the `\eqref` into a link, which does not seem to work within reveal.js - it just jumps to the first slide.

**Is there easy any way that such links could be corrected to point to the slide on which the equation appears?**

Failing that, does anyone know how to stop mathjax from creating these hyperlinks? Cleary it is better to have no hyperlink rather than a broken one.

Note, in order to get auto-numbering working, I have the following mathjax config lines to my html file, in addition to including the math plugin
````html
<script type="text/x-mathjax-config">
    MathJax.Hub.Config({
		TeX: { equationNumbers: { autoNumber: "AMS" } },
    });
</script>

````

**Comment by ndaman** [comment]
I am having the same issue (although with manual numbering and using \ref instead of \eqref), and would appreciate any advice.

For reference, although I write my equations in an external markdown file, I can see by inspecting elements that there is an "<mtd id='mjx-eqn-eqref'>" generated, which the reference tries to link to using "<mrow class='MathJax_ref' href='#mjx-eqn-eqref'>"

As nearly as I can tell, everything matches the correct syntax for an internal link, so why does it jump to the first slide instead of the correct one?

**Comment by chnchg** [comment]
I am using MathJax 3.1 and work around this problem by using the [tagformat](https://docs.mathjax.org/en/v3.1-latest/input/tex/extensions/tagformat.html) extension. The following will slide to the page of the referred equation upon click.
```javascript
tagformat:{
    url:(id,base)=>{
        var i = Reveal.getIndices(document.getElementById(id).closest('section'));
        return 'javascript:Reveal.slide('+i.h+','+i.v+')';
    }
}
```
You can turn off the link by returning an empty string `''`.
